e2e/test/scenarios/models/reproductions/25884-34349-null-description-in-column-metadata.cy.spec.ts

```diff
@@ -0,0 +1,45 @@
+import { SAMPLE_DATABASE } from "e2e/support/cypress_sample_database";
+import { openQuestionActions, popover, restore } from "e2e/support/helpers";
+
+const { ORDERS, ORDERS_ID } = SAMPLE_DATABASE;
+
+const ID_DESCRIPTION =
+  "This is a unique ID for the product. It is also called the “Invoice number” or “Confirmation number” in customer facing emails and screens.";
+
+describe("issues 25884 and 34349", () => {
+  beforeEach(() => {
+    restore();
+    cy.signInAsAdmin();
+  });
+
+  it("should show empty description input for columns without description in metadata (metabase#25884, metabase#34349)", () => {
+    cy.createQuestion(
+      {
+        type: "model",
+        query: {
+          "source-table": ORDERS_ID,
+          expressions: {
+            Country: ["substring", "United States", 1, 20],
+          },
+          fields: [
+            ["field", ORDERS.ID, { "base-type": "type/BigInteger" }],
+            ["expression", "Country", { "base-type": "type/Text" }],
```

Comment:
Using only 2 columns for this model so that we don't have to scroll the table horizontally to click the custom column header (which is the last column). Scrolling the table with `scrollTo("right")` didn't work very well, we'd have to call it twice or after a delay.